### PR TITLE
for issue 2294

### PR DESCRIPTION
**Pull request by sumitdaga**

### CODE REVIEW

File: src/shared/components/Settings/Account/MyAccount/index.jsx [hunk 1]
```diff
@@ -696,9 +682,9 @@ export default class MyAccount extends React.Component {
                           <div styleName="field col-2">
                             <div styleName="validation-bar" className="form-field">
                               <div styleName={`password toggle-password ${focus['re-new-password-input'] ? 'focus' : ''}`}>
-                                <input id="re-new-password-input" styleName="password-input" ref={this.reNewPasswordRef} onChange={this.checkPassword} onFocus={this.onPasswordFocus} onBlur={this.onPasswordBlur} name="re-password" type={passwordInputType['new-password-input']} placeholder="TYPE YOUR NEW PASSWORD" minLength="8" maxLength="64" required />
+                                <input id="re-new-password-input" styleName="password-input" ref={this.reNewPasswordRef} onChange={this.checkPassword} onFocus={this.onPasswordFocus} onBlur={this.onPasswordBlur} name="re-password" type={passwordInputType['re-new-password-input']} placeholder="TYPE YOUR NEW PASSWORD" minLength="8" maxLength="64" required />
                                 <label htmlFor="newPasswordCheckbox" styleName="passwordCheckbox">
```
Comment (ThomasKranitsas) [inline src/shared/components/Settings/Account/MyAccount/index.jsx]: Might want to change the `htmlFor` as well?

Comment (ThomasKranitsas) [reply src/shared/components/Settings/Account/MyAccount/index.jsx]: @sumitdaga

Comment (sumitdaga) [reply src/shared/components/Settings/Account/MyAccount/index.jsx]: @ThomasKranitsas  
oh yes i was meaning to but forgot ...i have done it now!
but any idea why it is not working ?

Comment (ThomasKranitsas) [reply src/shared/components/Settings/Account/MyAccount/index.jsx]: what do you mean is not working?

Comment (sumitdaga) [reply src/shared/components/Settings/Account/MyAccount/index.jsx]: when you click on the label the respective input element should focus right ?
it does not ...not just on this page but on other pages too ..  
for ex Add new device form in Tools > Devices

**Pull request merged**